README.md

```diff
@@ -20,11 +18,10 @@ Or install it yourself as:
 
     $ gem install htmx-rails
 
-After installing the gem, add this to your `app/assets/javascripts/application.js`:
+After installing the gem, run the installer:
+
+    $ rails g htmx_rails:install
```

Comment:
can it be `rails g htmx-rails:install` ?

Reply:
If the generator is in `Htmx::Rails::Generators` modules namespaces, the command will be `rails g htmx:rails:install`.

Maybe we can use `Htmx::Generators` module, so the resulting command will be `rails g htmx:install`

Reply:
sounds good to me